### ISSUE TITLE
Bot doesn't check whether it can actually join a vc

### ISSUE THREAD

**Issue by lolpoe** [brblacky/lavamusic#50]
**Describe the bug**
Bot doesn't check whether it's in a vc / can actually join a vc

**To Reproduce**
1. Play music in a vc where the bot has **permission** to join
2. Leave the vc
3. Manually disconnect bot or delete the channel
4. Join a vc the bot **doesn't** have **permission** to join
![image](https://user-images.githubusercontent.com/93990656/143985100-51029f48-2bab-4743-8ac6-6f977d99edfa.png)
5. Type join command 
6. The bot says 
![image](https://user-images.githubusercontent.com/93990656/143985066-e6faf161-2c85-453d-ad28-7769e4bad335.png)
but **doesn't join vc**. It still shows now playing and lets you queue songs.

**Expected behavior**
Bot should say it is unable to join the vc and not start a player

**Note**
When this errror occurs the only way to fix is restart entire bot

https://user-images.githubusercontent.com/93990656/143985666-724525d2-fa31-447d-a7ab-82a075a9412b.mp4

**Comment by brblacky** [comment]
Just add vc permission in join command

**Comment by brblacky** [comment]
```js
if(!message.guild.me.permissions.has([Permissions.FLAGS.CONNECT, Permissions.FLAGS.SPEAK])) return message.channel.send(`I don't have enough permissions to execute this command.`);
```

**Comment by lolpoe** [comment]
> Just add vc permission in join command

Um this is a pretty serious issue why not just implement fix. Adding this 1 line of code is just returning a message.
The same thing is going to happen. 

https://user-images.githubusercontent.com/93990656/145312932-e571a4fd-d413-45a7-8167-797f9fbcaca9.mp4

**Comment by brblacky** [comment]
It is fixed kindly check this https://github.com/brblacky/lavamusic/commit/af57c05ea52f112d5a268b83d169cedf32d3c286

**Comment by lolpoe** [comment]
> It is fixed kindly check this [af57c05](https://github.com/brblacky/lavamusic/commit/af57c05ea52f112d5a268b83d169cedf32d3c286)

![image](https://user-images.githubusercontent.com/93990656/145395098-1e9fa794-3f90-4bee-955f-1fb502c582cb.png)
![image](https://user-images.githubusercontent.com/93990656/145395118-0bb207a3-1bdc-4c96-b53c-a4a194000b6d.png)

still creates player
